app/models/vm_synchronize.rb

```diff
@@ -18,7 +18,7 @@ def call_synchronize
 
     options[:categories] ||= Vm.default_scan_categories
     begin
-      host = Host.find(agent_id)
+      host = MiqServer.find(agent_id)
```

Comment:
It's hard to understand why this would not be blowing up. Perhaps this code has not been used for a long time. @roliveri can you confirm?